app/controllers/steps/capital/property_address_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Steps
+  module Capital
+    class PropertyAddressController < Steps::CapitalStepController
+      def edit
+        @form_object = PropertyAddressForm.build(
+          property_record, crime_application: current_crime_application
+        )
+      end
+
+      def update
+        update_and_advance(PropertyAddressForm, record: property_record, as: :property_address)
+      end
+
+      private
+
+      def property_record
+        @property_record ||= properties.find(params[:property_id])
+      rescue ActiveRecord::RecordNotFound
+        raise Errors::PropertyNotFound
+      end
+
+      def properties
+        @properties ||= current_crime_application.properties
+      end
```

Comment:
Is there a need to memoize `properties` I think it is only used once on Line 17?